### DISCUSSION TITLE
obj.toActiveSelection() is not working after the 6.0 update

### DISCUSSION THREAD

**Discussion by sohailahmad07** [fabricjs/fabric.js#9969] [Q&A] [answered]
Hi so before when you want to ungroup an object you can use `obj.toActiveSelection()` but after 6.0 its is not woking and on the docs i does not method any alternative any ideas on how to ungroup objects

**Comment by asturur** [comment, marked as answer]
This was mentioned that was removed in a PR update, i missed it, but i want to add it back

**Reply by sohailahmad07** [reply]
k let know when its added

**Reply by asturur** [reply]
you will see it from the changelog
